Dockerfile

```diff
@@ -0,0 +1,56 @@
+# Start with a base image
+FROM python:3-onbuild as base
+# FROM apache/airflow:master-python3.6-ci
+# FROM python:3.6-slim-buster
+# FROM continuumio/miniconda3
+
+ARG DB_HOST=localhost
+ARG DB_PORT=5432
+ARG DB_NAME=postgres
+ARG DB_USER=postgres
+ARG DB_PASS
+
+ENV orion_prod=postgres+psycopg2://${DB_USER}:${DB_PASS}@${DB_HOST}:${DB_PORT}/${DB_NAME}
+ENV orion_test=postgres+psycopg2://${DB_USER}:${DB_PASS}@${DB_HOST}:${DB_PORT}/postgres
+ENV airflow_db=postgres+psycopg2://${DB_USER}:${DB_PASS}@${DB_HOST}:${DB_PORT}/airflow
```

Comment:
Added a separate db for airflow. I had to create it on AWS first. This is also passed on to `AIRFLOW__CORE__SQL_ALCHEMY_CONN`.